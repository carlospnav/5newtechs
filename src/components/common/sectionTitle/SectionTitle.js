import styled from "styled-components"
import css from "theme/variables"

const Title =  styled.h4`
  font-family: ${css.font.roboto};
  color: ${css.color.sectionTitle};
  width: 100%;
  margin: 0 0 20px 0;
`

export default function({children}){
  return (
    <Title>{children}</Title>
  )
}