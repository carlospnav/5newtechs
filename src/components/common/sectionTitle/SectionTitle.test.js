import { shallow } from "enzyme"
import SectionTitle from "./SectionTitle"
import "jest-styled-components"

describe("<SectionTitle>", () => {
  it("should render a SectionTitle properly.", () => {
    const wrapper = shallow(<SectionTitle />)
    expect(wrapper).toMatchSnapshot()
  })
})