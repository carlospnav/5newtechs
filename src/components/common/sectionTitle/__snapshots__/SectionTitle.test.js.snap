// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionTitle> should render a SectionTitle properly. 1`] = `
.c0 {
  font-family: 'Roboto',sans-serif;
  color: #4e5760;
  width: 100%;
  margin: 0 0 20px 0;
}

<h4
  className="c0"
/>
`;
